src/pages/Settings/SavedSearchBeta.vue

```diff
@@ -68,3 +75,28 @@ export default {
 	}
 };
 </script>
+
+<style scoped>
```

Comment:
Try to replace as much of this as possible with tailwind.

Reply:
Agree all of these styles should be converted to `tw-` classes.

Except for the 10k...please remove that everywhere you have it.